.travis.yml

```diff
@@ -144,6 +166,13 @@ install:
     # Install matplotlib
     pip install -ve .
 
+before_script:
+  - |
+    if [[ $TRAVIS_OS_NAME != 'osx' ]]; then
+      export DISPLAY=:99.0
+      sh -e /etc/init.d/xvfb start
+    fi
```

Comment:
Except that we're going to build matplotlib before running any tests, so xvfb should have more than enough time to start :-)